src/components/Cart.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react' // eslint-disable-line no-unused-vars
+import { ListGroup, Card, Button } from 'react-bootstrap' // eslint-disable-line no-unused-vars
+import CartItem from './CartItem'
+import { SessionContext } from '../App'
+import StripeCheckout from 'react-stripe-checkout'
+
+function Cart () {
+  const { state, dispatch } = React.useContext(SessionContext)
+
+  function handleToken(token, addresses) { // eslint-disable-line no-unused-vars
+    // handle paying process
+    handlePurchasePokemons()
+  }
+
+  function handlePurchasePokemons() {
+    dispatch({ type: 'PURCHASE_POKEMONS' })
+  }
+
+  return (
+    <>
```

Comment:
Why are you using a fragment if you already have a single child on it?

Reply:
I think it was because I had more than one element inside, but I didn't realize that I don't need them anymore. 
Thanks, I'll address that